lib/v2/datasource/remote/model/token_model.dart

```diff
@@ -21,44 +22,54 @@ class TokenModel extends Equatable {
     required this.backgroundImage,
     required this.logo,
     required this.balanceSubTitle,
+    this.precision = 4,
   });
 
   @override
   List<Object?> get props => [chainName, contract, symbol];
+
+  String getAssetString(double quantity) {
+    return "${quantity.toStringAsFixed(precision)} $symbol";
+  }
 }
 
 const SeedsToken = TokenModel(
-    chainName: "Telos",
-    contract: "token.seeds",
-    symbol: "SEEDS",
-    name: "Seeds",
-    backgroundImage: 'assets/images/wallet/currency_info_cards/seeds/background.jpg',
-    logo: 'assets/images/wallet/currency_info_cards/seeds/logo.jpg',
-    balanceSubTitle: 'Wallet Balance');
+  chainName: "Telos",
+  contract: "token.seeds",
+  symbol: "SEEDS",
+  name: "Seeds",
+  backgroundImage: 'assets/images/wallet/currency_info_cards/seeds/background.jpg',
+  logo: 'assets/images/wallet/currency_info_cards/seeds/logo.jpg',
+  balanceSubTitle: 'Wallet Balance',
+);
 
 const HusdToken = TokenModel(
-    chainName: "Telos",
-    contract: "husd.hypha",
-    symbol: "HUSD",
-    name: "HUSD",
-    backgroundImage: 'assets/images/wallet/currency_info_cards/hypha/background.jpg',
-    logo: 'assets/images/wallet/currency_info_cards/hypha/logo.jpg',
-    balanceSubTitle: 'Wallet Balance');
+  chainName: "Telos",
+  contract: "husd.hypha",
+  symbol: "HUSD",
+  name: "HUSD",
+  backgroundImage: 'assets/images/wallet/currency_info_cards/hypha/background.jpg',
+  logo: 'assets/images/wallet/currency_info_cards/hypha/logo.jpg',
+  balanceSubTitle: 'Wallet Balance',
+  precision: 2,
+);
```

Comment:
most is formatting here, but HUSD has a precision of 2, not 4, so overriding this here